standards/nist_secure-software-development-framework_1.1.json

```diff
@@ -0,0 +1,562 @@
+          },
+          {
+            "bom-ref": "ssdf-1.1-RV.2.2",
+            "identifier": "RV.2.2",
+            "text": "Plan and implement risk responses for vulnerabilities.",
+            "opencre": null,
+            "parent": "-RV.2",
+            "externalReferences:": []
+          },
+          {
+            "bom-ref": "ssdf-1.1-RV.3",
+            "identifier": "RV.3",
+            "text": "Help reduce the frequency of vulnerabilities in the future.",
+            "opencre": null,
+            "parent": "",
+            "externalReferences:": []
+          },
+
+          {
+            "bom-ref": "ssdf-1.1-RV.3.1",
+            "identifier": "RV.3.1",
+            "text": "Analyze identified vulnerabilities to determine their root causes.",
+            "opencre": null,
+            "parent": "-RV.3",
+            "externalReferences:": []
+          },
+          {
+            "bom-ref": "ssdf-1.1-RV.3.2",
+            "identifier": "RV.3.2",
+            "text": "Analyze the root causes over time to identify patterns, such as a particular secure coding practice not being followed consistently.",
+            "opencre": null,
+            "parent": "-RV.3",
+            "externalReferences:": []
+          },
+          {
+            "bom-ref": "ssdf-1.1-RV.3.3",
+            "identifier": "RV.3.3",
+            "text": "Review the software for similar vulnerabilities to eradicate a class of vulnerabilities, and proactively fix them rather than waiting for external reports.",
+            "opencre": null,
+            "parent": "-RV.3",
+            "externalReferences:": []
+          },
+          {
+            "bom-ref": "ssdf-1.1-RV.3.4",
+            "identifier": "RV.3.4",
+            "text": "Review the SDLC process, and update it if appropriate to prevent (or reduce the likelihood of) the root cause recurring in updates to the software or in new software that is created.",
+            "opencre": null,
+            "parent": "-RV.3",
+            "externalReferences:": []
+          }
+        ],
+        "externalReferences:": [
+          {
+            "type": "documentation",
+            "url": "https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-218.pdf",
+            "comment": "NIST SP 800-218",
+            "hashes": [
+              {
+                "alg": "SHA3-256",
+                "content": "88b0683941e1fb5900dc2723555321320520c9d9075b4c82163545992d6389fd"
+              }
+            ]
+          },
+          {
+            "type": "documentation",
+            "url": "https://csrc.nist.gov/csrc/media/Publications/sp/800-218/final/documents/NIST.SP.800-218.SSDF-table.xlsx",
+            "comment": "NIST SP 800-218 in excel",
+            "hashes": [
+              {
+                "alg": "SHA3-256",
+                "content": "260d88cf62c8742116a7be95e9257848e836eb11ecd8779b0099efd3a25422e7"
+              }
+            ]
+          }
+        ],
+        "signature": {}
```

Comment:
Should we sign this somehow as the generator? If not, should this not be here? Or do we want it present even if empty to indicate explicitly that it is not signed?